resources/migrations/001_update_migrations.yaml

```diff
@@ -8103,6 +8103,7 @@ databaseChangeLog:
       id: v50.2024-06-12T12:33:08
       author: piranha
       comment: 'Copy old cache configurations to cache_config table'
+      validCheckSum: ANY
```

Comment:
I use any but not an exact checksum here because apparently liquibase output different checksum for different dbms because it's based on the content of each SQL File.